defaults.h

```diff
@@ -109,6 +109,7 @@
   #define DEFAULT_DECAY_MODE 0         //slowest
   #define DEFAULT_COUNTS_PER_IDX 4000  //counts per encoder rev
   #define DEFAULT_FORCE_SENSOR_LEVEL 77 // 1.5V out of 5.0V=255 max
+  #define DEFAULT_MAG_GAP_LIMIT 12.5 //in mm (units)
```

Comment:
We might want to consider over-writing this via a EEPROM setting.  I could see this as something we might want to configure.  Just a though.